src/main/resources/application.yaml

```diff
@@ -66,3 +66,13 @@ spring:
 azure:
   application-insights:
     instrumentation-key: ${darts.AppInsightsInstrumentationKey:00000000-0000-0000-0000-000000000000}
+
+darts:
+  notification:
+    max_retry_attempts: 3
+    gov-notify:
+      api-key: ${darts.GovNotifyApiKey:00000000-0000-0000-0000-000000000000}
+      template-map:
+        court_manager_approve_transcript: 2f9e18b5-5046-44c2-8abd-e4ff15008a60
+        request_to_transcriber: 976bf288-705d-4cbb-b24f-c5529abf14cf
+        requested_audio_is_available: 66a1864f-24a6-469a-ac55-66bc57c7e4f6
```

Comment:
Maybe we could future-proof this incase multiple notifications are required for an event, something like

```
darts:
  notification:
    max_retry_attempts: 3
    gov-notify
      court_manager_approve_transcript:
        emailTemplateId: 2f9e18b5-5046-44c2-8abd-e4ff15008a60
        smsTemplateId: [we can add this is we wanted]
      request_to_transcriber:
        emailTemplateId: 976bf288-705d-4cbb-b24f-c5529abf14cf
      requested_audio_is_available:
        emailTemplateId: 66a1864f-24a6-469a-ac55-66bc57c7e4f6
```

Reply:
nevermind, it can be easily done

Reply:
as discussed, introducing sms functionality, would involve db changes, and so is out of scope of the current ticket.